motorway/contrib/amazon_sqs/intersections.py

```diff
@@ -1,5 +1,6 @@
 from motorway.intersection import Intersection
 import boto3
+from botocore.exceptions import Que, ClientError
```

Comment:
`Que` imported but not used, is there really a class Que? :D

Reply:
Hmm, this looks like a leftover, good catch

Reply:
Removed